templates/experiments/experiment_form.html

```diff
@@ -122,6 +122,17 @@ <h3 class="text-md font-bold">{% translate "Please note" %}</h3>
       </div>
     </div>
 {% endblock form %}
+{% block form_actions %}
+  <input type="submit" class="pg-button-primary mt-2" value="{{ button_text }}">
```

Comment:
the `mt-2` on this is going to put the alignment out - maybe wrap them all in a div with `mt-2`